web-ts/tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
+    "target": "ES2020",
```

Comment:
This one we could discuss. On `IGAPP-web-ts-init` it was set to esnext, which is at least in my opinion not a stable form of release. In our current web project we use es6(=es2015) syntax.

Reply:
I think `ES2020` is fine for now :+1:

Reply:
I don't think that this config will support IE11, but I'd just leave this and see some time later what to choose here.

Reply:
I see what you mean here. Probably we will have to stay with es6, as previously. Check out [Caniuse.com](https://caniuse.com/?search=es6).